scripts/stack/README.md

```diff
@@ -0,0 +1,106 @@
+# Full Elastic Stack with Connectors
+
+We know that setting up the Connectors service may look a bit daunting, but to help with this, we pulled together a set of scripts to enable a user to get up and running with a full stack with connectors. Here, we provide an example configuration, Docker Compose configuration, and a bash script to startup and run a stack including Elasticsearch, Kibana, and Connectors.
+
+These scripts can be useful if you want to try out Connectors and ingesting their data into an Elasticsearch instance, or, as a starting place for adding Connectors to your production environment.
+
+# Contents
+
+* [Prerequisites](#prerequisites)
+* [Running the Stack](#running-the-stack)
+* [Prompted Configuration](#prompted-configuration)
+* [Manual Configuration](#manual-configuration)
+* [Script command reference](#script-command-reference)
+  * [run-stack.sh](#run-stacksh)
+  * [stop-stack.sh](#stop-stacksh)
+
+## Prerequisites
+* Linux or MacOS (Windows is not currently supported)
```

Comment:
Curious - why Windows not supported? I think some ppl run connectors container on windows

Reply:
Windows is not supported for running these particular scripts, but it is for the Connectors service... I'll clarify this. Thanks!